test/webdriverio/test/basic_test.ts

```diff
@@ -93,48 +94,350 @@ suite('Keyboard navigation', function () {
+    await this.browser.keys(Key.ArrowLeft);
+    await this.browser.pause(PAUSE_TIME);
+
+    chai.assert.equal(
+      await getCurrentFocusNodeId(this.browser),
+      'math_modulo_1',
+    );
+  });
+
+  test('Right from first inline input selects second inline input', async function () {
+    await focusWorkspace(this.browser);
+    await this.browser.pause(PAUSE_TIME);
+    await setCurrentCursorNodeById(this.browser, 'math_number_2');
+    await this.browser.pause(PAUSE_TIME);
+    await this.browser.keys(Key.ArrowRight);
+    await this.browser.pause(PAUSE_TIME);
+
+    chai.assert.equal(
+      await getCurrentFocusNodeId(this.browser),
+      'math_number_3',
+    );
+  });
+
+  test('Left from second inline input selects first inline input', async function () {
+    await focusWorkspace(this.browser);
+    await this.browser.pause(PAUSE_TIME);
+    await setCurrentCursorNodeById(this.browser, 'math_number_3');
+    await this.browser.pause(PAUSE_TIME);
+    await this.browser.keys(Key.ArrowLeft);
+    await this.browser.pause(PAUSE_TIME);
+
+    chai.assert.equal(
+      await getCurrentFocusNodeId(this.browser),
+      'math_number_2',
+    );
+  });
+
+  test("Right from last inline input selects block's next connection", async function () {
+    await focusWorkspace(this.browser);
+    await this.browser.pause(PAUSE_TIME);
+    await setCurrentCursorNodeById(this.browser, 'colour_picker_1');
+    await this.browser.pause(PAUSE_TIME);
+    await this.browser.keys(Key.ArrowRight);
+    await this.browser.pause(PAUSE_TIME);
+
+    chai
+      .expect(await getCurrentFocusNodeId(this.browser))
+      .to.include('simple_circle_1_connection_');
+
+    chai.assert.equal(
+      await getFocusedConnectionType(this.browser),
+      Blockly.ConnectionType.NEXT_STATEMENT,
+    );
+  });
+
+  test("Down from inline input selects block's next connection", async function () {
+    await focusWorkspace(this.browser);
+    await this.browser.pause(PAUSE_TIME);
+    await setCurrentCursorNodeById(this.browser, 'colour_picker_1');
+    await this.browser.pause(PAUSE_TIME);
+    await this.browser.keys(Key.ArrowDown);
+    await this.browser.pause(PAUSE_TIME);
+
+    chai
+      .expect(await getCurrentFocusNodeId(this.browser))
+      .to.include('simple_circle_1_connection_');
+
+    chai.assert.equal(
+      await getFocusedConnectionType(this.browser),
+      Blockly.ConnectionType.NEXT_STATEMENT,
+    );
+  });
+
+  test("Down from inline input selects block's child connection", async function () {
+    await focusWorkspace(this.browser);
+    await this.browser.pause(PAUSE_TIME);
+    await setCurrentCursorNodeById(this.browser, 'math_number_2');
+    await this.browser.pause(PAUSE_TIME);
+    await this.browser.keys(Key.ArrowDown);
+    await this.browser.pause(PAUSE_TIME);
+
+    chai
+      .expect(await getCurrentFocusNodeId(this.browser))
+      .to.include('controls_repeat_ext_1_connection_');
+  });
+
+  /*
```

Comment:
Use `test.skip` instead of commenting it out. Also file an issue to come back to these, as discussed, and add a TODO(issue number) here.

Reply:
Done.